controllers/playersIndexController.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../db');
+
+// GET http://localhost:7000/admin/players
+exports.getIndex = (req, res, next) => {
+    console.log("GET: Hello getPlayersPage")
+    // connect to the database and save the new incoming player
+    db.getDb()
+        .db('softball')
+        // the collction will be created dynamically if it does not exist yet
+        .collection('players')
+        .find(
+            {},
+            {_id: 0, name: 1, email: 1}
+        )
+        .toArray(
+            function(err, result) {
+                if (err) throw err;
+                if (result) {
+                    res
+                    .status(200)
+                    .json(result)
+                } else {
+                    res.send(JSON.stringify({
```

Comment:
let's send some error code. `500` (Internal server error) would be good here. and more descriptive error message like "db-find-error". So the front end engineer can have a better idea about the error.

Reply:
![Screenshot 2021-09-07 at 17 13 37](https://user-images.githubusercontent.com/60405643/132369163-67a849fe-44ff-482d-b673-4d325d514c4b.png)

Reply:
`.status(500)`